scripts/breaking_changes_checker/detect_breaking_changes.py

```diff
@@ -469,6 +477,29 @@ def remove_json_files(pkg_dir: str) -> None:
     _LOGGER.info("cleaning up")
 
 
+def report_azure_mgmt_versioned_module(code_report):
+    
+    def parse_module_name(module):
+        split_module = module.split(".")
+        # Azure mgmt packages are typically in the form of: azure.mgmt.<service>
+        # If the module has a version, it will be in the form of: azure.mgmt.<service>.<version> or azure.mgmt.<service>.<version>.<submodule>
+        if len(split_module) >= 4:
```

Comment:
nit: there is a special multi-api package https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/resources/azure-mgmt-resource/azure/mgmt/resource which contains multi modules and each module has many sub api-version folders. It equals that put several multi-api package into one package.

Reply:
done, looping instead